tests/run_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source env/bin/activate
```

Comment:
The python virtual env is created as `venv` in the config script.

```suggestion
source venv/bin/activate
```